node/config_opts.go

```diff
@@ -18,10 +18,10 @@ func WithTrustedHash(hash string) Option {
 	}
 }
 
-// WithTrustedPeer appends TrustedPeer to the Config.
-func WithTrustedPeer(addr string) Option {
+// WithTrustedPeers appends a slice of TrustedPeers to the Config.
+func WithTrustedPeers(addr []string) Option {
```

Comment:
```suggestion
// WithTrustedPeers appends new "trusted peers" to the Config.
func WithTrustedPeers(addr ...string) Option {
```

Reply:
This way you won't need to wrap values with an array and simply pass them in options as many as you want

Reply:
Done